dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpServiceContextImpl.java

```diff
@@ -3686,6 +3686,9 @@ private void nettyWrite(boolean sendHeaders, boolean finalWrite) throws IOExcept
 
             getTSC().getWriteInterface().setBuffers(writeBuffers);
             try {
+                if(!nettyContext.channel().isOpen()){
+                    throw new IOException("Attempted to write on a closed Netty channel");
```

Comment:
I'm testing this locally but does this end up in the logs? If so we need an issue to ensure it is translated.

Reply:
This would be the equivalent to a broken pipe exception from the JDK used in legacy. The exception shows up on the logs but I don't think it's worth translating since we don't do so for the equivalent in legacy

Reply:
We can track it using this [issue](https://github.com/orgs/OpenLiberty/projects/39?pane=issue&itemId=114149390&issue=OpenLiberty%7Copen-liberty%7C31761)